website/versioned_docs/version-2.0/fdc3-charter.md

```diff
@@ -40,7 +40,7 @@ To be successful, the maintenance and evolution of this Standard needs to have a
 ## Licensing
 Version 1.0 of the FDC3 specification is licensed under the [FDC3 1.0 Final Specification License](https://github.com/finos/FDC3/blob/17892008c26a73ff1fd9f6e40ceb8c8bfd58c610/PATENTS-FDC3-1.0.md).
```

Comment:
Should this link to patents file on head?

Reply:
no, this is for the 2.0 version of FDC3, which is using the old license.